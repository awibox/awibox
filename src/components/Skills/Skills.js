import React from 'react';
import PropTypes from 'prop-types';
// Styles
import styles from './Skills.scss';

const Skills = (props) => (
  <div className={styles.skills}>
    {props.data.map((item) => (
      <div className={styles.skillItem} key={item}>{item}</div>
    ))}
  </div>
);

Skills.propTypes = {
  data: PropTypes.array,
  className: PropTypes.string,
};

export default Skills;
