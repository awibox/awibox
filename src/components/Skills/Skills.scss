@import '../../styles/variables';

.skills {

}

.skillItem {
  background-color: #f3f3f4;
  padding: 5px 12px;
  color: inherit;
  border-radius: 2px;
  border: 1px solid #e7eaec;
  margin-right: 5px;
  margin-bottom: 5px;
  display: inline-flex;
}
