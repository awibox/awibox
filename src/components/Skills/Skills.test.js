import React from 'react';
import { shallow } from 'enzyme';
import { List } from 'immutable';
import Skills from './Skills';

const mockSkills = List([
  { name: 'react' },
  { name: 'js' },
  { name: 'css' },
]);

describe('Card', () => {
  it('should render correctly', () => {
    const component = shallow(<Skills data={mockSkills} />);
    expect(component).toMatchSnapshot();
  });
});
