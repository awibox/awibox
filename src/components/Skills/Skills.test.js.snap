// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should render correctly 1`] = `
<div>
  <div
    className="skillItem"
    key="react"
  >
    react
  </div>
  <div
    className="skillItem"
    key="js"
  >
    js
  </div>
  <div
    className="skillItem"
    key="css"
  >
    css
  </div>
</div>
`;
